README.md

```diff
@@ -198,7 +198,7 @@ Sometimes, it's desirable to pass additional local variables to partials. It can
 
 `public_activity` looks for views in `app/views/public_activity`.
 
-For example, if you have an activity with `:key` set to `"activity.user.changed_avatar"`, the gem will look for a partial in `app/views/public_activity/user/_changed_avatar.(erb|haml|slim|something_else)`.
+For example, if you have an activity with `:key` set to `"activity.user.changed_avatar"`, the gem will look for a partial in `app/views/public_activity/user/_changed_avatar.(html|erb|haml|slim|something_else)`.
```

Comment:
I think the convention is `template.format.preprocessor`, like `_changed_avatar.html.haml` / `application.js.coffee`